bootstrap/cmd/bootstrap/app/options/options.go

```diff
@@ -50,4 +52,6 @@ func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&s.Email, "email", "", "Your Email address for GCP account, if you are using GKE.")
 	fs.StringVar(&s.IpName, "ip-name", "kubeflow", "Name of the ip you reserved on GCP project")
 	fs.BoolVar(&s.InCluster, "in-cluster", false, "Whether bootstrapper is executed inside a pod")
+	fs.StringVar(&s.RegistryUri, "registry-uri", "/opt/kubeflow/kubeflow", "Location of kubeflow registry.")
+	fs.BoolVar(&s.Testing, "testing", false, "Whether bootstrapper is executed for e2e test.")
```

Comment:
Why do we need an option in the bootstrapper about the E2E tests?

Reply:
When testing = true, bootstrapper will exit after setting up resources.